src/couch/src/couch_changes.erl

```diff
@@ -264,6 +264,26 @@ filter(Db, DocInfo, {custom, Style, Req0, DDoc, FName}) ->
     {ok, Passes} = couch_query_servers:filter_docs(Req, Db, DDoc, FName, Docs),
     filter_revs(Passes, Docs).
 
+filter(Db, DocInfo, {selector, Style, {Selector, _Fields}}, IncludeDocs) when IncludeDocs ->
```

Comment:
We could just use `true` for `IncludeDocs` in the function args and match directly on it.

Reply:
good idea!